common/functions/src/scalars/dates/week_date.rs

```diff
@@ -201,14 +201,22 @@ where
 
         let func = FunctionAdapter::create(func, true);
         let left_val = func
-            .eval(&[args[0].left.clone().unwrap()], 1, FunctionContext {
-                tz: None,
-            })?
+            .eval(
+                FunctionContext {
```

Comment:
why not pass the `_func_ctx` to func?

Reply:
Oh. Miss It. Will be fixed

Reply:
`timezone` will not affect the function's monotonicity, and we do not have `FunctionContext` parameter in `get_monotonicity` func. 
so maybe a default value will be ok. @sundy-li